ios/RCTWebRTC/WebRTCModule+RTCPeerConnection.m

```diff
@@ -321,6 +445,37 @@ @implementation WebRTCModule (RTCPeerConnection)
   }];
 }
 
+RCT_EXPORT_METHOD(getTrackVolumes:(RCTResponseSenderBlock)callback)
+{
+    RTCMediaStreamTrack *track = nil;
+    __block int statsRemaining = self.peerConnections.count;
+    __block NSMutableArray *statsAll = [NSMutableArray new];
+
+    for(id key in self.peerConnections) {
+        RTCPeerConnection *peerConnection = self.peerConnections[key];
+
+        [peerConnection statsForTrack:track statsOutputLevel:RTCStatsOutputLevelStandard completionHandler:^(NSArray<RTCLegacyStatsReport *> *stats) {
```

Comment:
react-native-webrtc has since started using the new stats (the original work for this PR was done early last year before it switched). It would be useful if this also utilised the new stats.

Reply:
@roblofthouse I've applied your changes expect this one on a local branch, because I can't seem to find any resources on how to migrate this. I also no longer own a mac so making this quick hard to test on my branch. But if you can point me to something I'll do the legwork